src/detail-page/DetailPage.js

```diff
@@ -33,7 +31,11 @@ const DetailSectionTitle = styled.h1`
   font-size: 20px;
   margin-bottom: 16px;
   margin-right: 32px;
-  width: ${sectionTextWidth}px;
+  width: ${(props) => props.theme.sectionTextWidthNormal}px;
+
+  .RaceNarrativePage & {
```

Comment:
Nit:  This `DetailPage` component is otherwise generic / agnostic to the particular page that it is being used within so maybe we should just call this class something like `WideSection` (or whatever) so it more readily lends itself to reuse if we add more dynamic text pages in the future.